Symfony/CS/Tests/FixerTest.php

```diff
@@ -184,10 +184,13 @@ public function getFixersPriorityCases()
             array($fixers['double_arrow_multiline_whitespaces'], $fixers['align_double_arrow']),
             array($fixers['indentation'], $fixers['phpdoc_indent']),
             array($fixers['phpdoc_order'], $fixers['phpdoc_separation']),
+            array($fixers['phpdoc_no_access'], $fixers['phpdoc_separation']),
+            array($fixers['phpdoc_no_access'], $fixers['phpdoc_order']),
             array($fixers['phpdoc_no_empty_return'], $fixers['phpdoc_separation']),
             array($fixers['phpdoc_no_empty_return'], $fixers['phpdoc_order']),
             array($fixers['phpdoc_no_package'], $fixers['phpdoc_separation']),
             array($fixers['phpdoc_no_package'], $fixers['phpdoc_order']),
+            array($fixers['phpdoc_no_access'], $fixers['phpdoc_trim']),
```

Comment:
Consider the following:

```php
/**
 * Foo!
  *
  * @access public
  */
```

If we remove the annotation, the next line above it would be caught by the phpdoc_trim fixer.